oxidation/libparsec/crates/platform_local_db/src/lib.rs

```diff
@@ -1,20 +1,16 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 //! Manage a local database connection.
 
-#![warn(clippy::missing_docs_in_private_items)]
```

Comment:
Keep it, 

1. If think it's nice the have the function with a little comment above them.
2. You have removed comment from functions who had it in the past.

Reply:
I disagree with this: comment should be used to make the code readable, not to please a linter

Commenting a component is important when it is exposed as part of an API (especially when it is a lib aimed at 3rd party)
When a component is only private, a good naming is often enough.

On top of that a global comment to explain how multiple components work together is better than splitting the explanation between the multiple involved components.

So I'd rather keep some liberty on how to document the code, to avoid the good old:
```rust
/// Ideo the Kojima
struct IdeoKojima {
   /// The Ideo Kojima is on Ideo mode
   Ideo,
   /// The Ideo Kojima is on Kojima mode
   Kojima,
}
```

Reply:
My discussion isn't about the linter :shrug: